index.html

```diff
@@ -42,5 +46,265 @@ <h1 class=" bigger-text">I'm Albert. Glad to see you!</h1>
+            <li class="back-end">
+              BACK END DEV
+            </li>
+            <li class="year">
+              2015
+            </li>
+            </ul>
+          </div>
+          <div class="second-small-text">
+            <p class="small">A daily selection of private personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div class="language">
+          <ul class="tags">
+            <li class="tag">
+              HTML
+            </li>
+            <li class="tag">
+              CSS
+            </li>
+            <li class="tag javascript">
+              JavaScript
+            </li>
+          </ul>
+          </div>
+          <div>
+            <button class="see-project-button" type="submit">
+            See project
+            </button>
+          </div>
+          </div>
+        </div>
+        </div>
+
+
+
+        <div class="first-phase">
+          <div class="phase-one">
+          <div>
+          <img src="image5.png" alt="image5" width="295" height="220">
+          </div>
+          <div class="block">
+            <div class="tonic">
+          <h1>
+           Multi-Post Stories
+          </h1>
+          </div>
+          <div class="second-text">
+            <span class="canopy">
+              CANOPY
+            </span>
+            <ul>
+            <li>
+              BACK END DEV
+            </li>
+            <li>
+              2015
+            </li>
+            </ul>
+          </div>
+          <div class="second-small-text">
+            <p class="small">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+          </div>
+          <div class="language">
+          <ul class="tags">
+            <li class="tag">
+              HTML
+            </li>
+            <li class="tag">
+              CSS
+            </li>
+            <li class="tag javascript">
+              JavaScript
+            </li>
+          </ul>
+          </div>
+          <div class="see-project">
+            <button class="see-project-button" type="submit">
+              See project
+            </button>
+          </div>
+          </div>
+          </div>
+        </div> 
+    </section>
+    <section>
+      <div class="about-container">
+          <div class="about-top">
+              <div>
+                  <h1>
+                      About Myself
+                  </h1>
+              </div>
+              <div>
+                  <p class="about-text">
+                      Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                      If you like what you see and have a project you need coded, don't hestiate to contact me.
+                  </p>
+              </div>
+              <div>
+                  <div class="connect">
+                      LET'S CONNECT
+                  </div>
```

Comment:
- [ ]  I would suggest you to use `h2` or `h3` for the heading `LET'S CONNECT` instead of wrapping in a `div` as it would be semantically more appropriate.